### PR TITLE
Add account api

### PR DESCRIPTION
**Pull request by EbenezerGH**
Blocked until #26 fixed.

### CODE REVIEW

File: etherscanapi/src/main/java/jfyg/queries/QueryMediator.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    override fun stats(module: String, action: String): Disposable? =
+            RestClient().getQuery()
+                    .getStats(module, action, ApiKey.takeOff.callApiKey())
+                    .observeOn(AndroidSchedulers.mainThread())
+                    .subscribeOn(Schedulers.io())
+                    .subscribe(this::handleResponse, this::handleError)
+
+
+    override fun handleResponse(response: AccountResponse) {
```
Comment (EbenezerGH) [inline etherscanapi/src/main/java/jfyg/queries/QueryMediator.kt]: This handleresponse() isn't being triggered.  The network call is executing correctly.. i've monitored the requests and the responses and they are going through fine according to charlesProxy..

File: README.md [hunk 2]
```diff
@@ -44,6 +44,14 @@ D/jfyg.etherscan.helloetherescan.MainActivity: The current price of Ether in Btc
 
 * [**Ebenezer Ackon**](http://www.ebenezerackon.com/)
 Only me so far but eagerly accepting help :)
+
+## Help me pay off my student loans
+```
+BTC: 1MLaf31sjVG3LbMyQEuY16qosPAhwNRQeZ
+ETH: 82e4499D4b2A669831a3881d61BB24f7b620c61a
+LTC: La1vhdzuVY85EEW8m5hTpw4C15gPY9hVDW
```
Comment (westonal) [inline README.md]: Still no VTC? :D

Comment (EbenezerGH) [reply README.md]: kekeke.. I will transfer some btc to vtc in abra.  I just never got around to it :D

Comment (westonal) [reply README.md]: you don't actually need any for an address :D

**Pull request merged**